src/python/pants/goal/anonymous_telemetry.py

```diff
@@ -146,15 +144,9 @@ def __call__(
                 # This is copied from humbug code, to ensure that future changes to humbug
                 # don't add tags that inadvertently violate our anonymity promise.
                 system_tags = [
-                    "humbug",
                     "source:{}".format(reporter.name),
                     "os:{}".format(reporter.system_information.os),
                     "arch:{}".format(reporter.system_information.machine),
-                    "python:{}".format(reporter.system_information.python_version_major),
```

Comment:
Why remove these? are they added automatically now?

Reply:
This is still sending `<major>.<minor>.<patch>`. We found that very few people are querying by the python versions at the moment, and we are adding functionality in Bugout search to query by simple expressions that would allow you to select major/major+minor version from the full tag.